src/core/style/style_change.cpp

```diff
@@ -49,6 +49,12 @@ std::vector<std::unique_ptr<StyleChange>> StyleChange::addParameter(const StyleP
         return changes;
     }
 
+    const auto *filterParameter = qobject_cast<const FilterParameter *>(parameter);
+    if (filterParameter != nullptr) {
+        changes.emplace_back(std::make_unique<StyleSetFilter>(filterParameter));
```

Comment:
This should just be `push_back` because this is also calling a move constructor.

Reply:
Fixed all cases.